finetune/utils/data/data_utils.py

```diff
@@ -36,7 +36,7 @@ def get_raw_dataset(dataset_name, output_path, seed, local_rank):
             and os.path.isfile(chat_path + "/data/eval.json")
         ):
             raise RuntimeError(
-                "Please check both the train.json and eval.json files in your local directory."
+                f"Please check both the train.json and eval.json files in your local directory."
```

Comment:
why f-string here?

Reply:
copy from a former version before going public and not refreshed by pull request
removed